style.css

```diff
@@ -0,0 +1,87 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Gemunu+Libre&family=Lato&display=swap');
+
+body{
+    margin:0;
+}
+
+html {
+    font-family: 'Lato', sans-serif;    
+}
+
+h1 {
+    font-family: 'Gemunu Libre', sans-serif;
+}
+
+.main-section h1{
+    font-size: 2rem;
+}
+
+code {
+    font-size: 1.2rem; 
+    background-color: rgba(95, 95, 95, 0.116);   
+}
+
+p, li {
+    font-size: 1.5rem;
+}
+
+#container {
+    height: 100vh;
+
+    width: 100%;
+    display: grid;
+    grid-template-areas: 'navbar documentation';
+    grid-template-columns: 20rem 1fr;
+    grid-template-rows: 1fr;
+}
+
+#navbar {
+    grid-template-rows: 100px repeat(auto-fit, 60px);
+    grid-area: navbar;
+    display: grid;
+    /* margin: 2px; */
+
+}
+
+#main-doc {
+    grid-area: documentation;
+    overflow: auto;
+    margin-left: 20px;
+    border-left: 5px solid black;
+    padding-left: 20px;
+    padding-bottom: 50px;
+}
+
+a{
+    box-sizing: border-box;
+    padding: 20px;
+    text-decoration: none;
+    font-size: 1.3rem;
+    color: black;
+    border-top: 2px solid rgb(0, 0, 0);
+
+}
+a:hover{
+    color: rgb(158, 11, 202);
+}
+
+nav header {
+    margin-left: 20px;
+    font-size: 1.2rem;    
+}
```

Comment:
Remove no needed space,  call the items inside their parents elements, and fix the indentation!

Reply:
- [ ] I think your comment should be more polite. So it should also be a suggestion instead of making it look like an order.  and please add the `WHY` for your requested changes 😄